src/fides/core/export.py

```diff
@@ -392,8 +400,12 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
                             )
                     cartesian_product_of_declaration.append(tuple(product))
 
-                output_list += cartesian_product_of_declaration
-        else:
+                declaration_rows += cartesian_product_of_declaration
+            output_list += declaration_rows
+
+        # if we haven't added any output rows for the system based on declarations,
+        # then add a "placholder" generla system row to ensure the system is captured in the output
```

Comment:
```suggestion
        # then add a "placeholder" general system row to ensure the system is captured in the output
```

Reply:
👍 fixed (rebased to keep commit history nice and simple)